Source/Core/Core/HotkeyManager.cpp

```diff
@@ -83,22 +83,22 @@ constexpr std::array<const char*, 133> s_hotkey_labels{{
     _trans("Toggle SD Card"),
     _trans("Toggle USB Keyboard"),
 
-    _trans("Next Profile for Wii Remote 1"),
-    _trans("Previous Profile for Wii Remote 1"),
-    _trans("Next Game Profile for Wii Remote 1"),
-    _trans("Previous Game Profile for Wii Remote 1"),
-    _trans("Next Profile for Wii Remote 2"),
-    _trans("Previous Profile for Wii Remote 2"),
-    _trans("Next Game Profile for Wii Remote 2"),
-    _trans("Previous Game Profile for Wii Remote 2"),
-    _trans("Next Profile for Wii Remote 3"),
-    _trans("Previous Profile for Wii Remote 3"),
-    _trans("Next Game Profile for Wii Remote 3"),
-    _trans("Previous Game Profile for Wii Remote 3"),
-    _trans("Next Profile for Wii Remote 4"),
-    _trans("Previous Profile for Wii Remote 4"),
-    _trans("Next Game Profile for Wii Remote 4"),
-    _trans("Previous Game Profile for Wii Remote 4"),
+    _trans("Next Profile"),
+    _trans("Previous Profile"),
+    _trans("Next Game Profile"),
+    _trans("Previous Game Profile"),
+    _trans("Next Profile "),
+    _trans("Previous Profile "),
+    _trans("Next Game Profile "),
+    _trans("Previous Game Profile "),
```

Comment:
Just noticed one more thing, there's extra spaces at the end of these strings. You can even see that in your screenshot.

Maybe also clean up your commit message while you're at it, all those extra fix lines are really not necessary.

Reply:
Done.